pkg/backend/filestate/backend.go

```diff
@@ -936,7 +936,7 @@ func (b *localBackend) Update(ctx context.Context, stack backend.Stack,
 func (b *localBackend) Import(ctx context.Context, stack backend.Stack,
 	op backend.UpdateOperation, imports []deploy.Import,
 ) (sdkDisplay.ResourceChanges, result.Result) {
-	err := b.Lock(ctx, stack.Ref())
+	err := b.Lock(ctx, stack.Ref(), 0)
```

Comment:
Shouldn't all these be op.Opts.LockTimeout as well?

Reply:
correct will fix that now